packages/core/discovery.config.default.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
       titleTemplate: '%s: Search results title',
       descriptionTemplate: '%s: Search results description',
       noIndex: true,
+      bodyH1: 'Showing results for:',
```

Comment:
This does the trick so far. For the long term, we should bring this from the hCMS search page settings.